rasa/core/training/story_reader/story_step_builder.py

```diff
@@ -57,14 +57,33 @@ def _prev_end_checkpoints(self) -> List[Checkpoint]:
             end_names = {e.name for s in self.current_steps for e in s.end_checkpoints}
             return [Checkpoint(name) for name in end_names]
 
-    def add_user_messages(self, messages: List[UserUttered]) -> None:
+    def add_user_messages(
+        self, messages: List[UserUttered], unfold_or_utterances: bool = True
+    ) -> None:
+        """Adds next story steps with the user's utterances.
+
+        Args:
+            messages: User utterances.
+            unfold_or_utterances: Identifies if the user utterance is a part of
+                                  OR statement. This parameter is used only to
+                                  simplify the conversation from MD story files.
+                                  Don't use it other ways, because it ends up
+                                  in a invalid story that cannot be user for
+                                  real training.
```

Comment:
```suggestion
            unfold_or_utterances: Identifies if the user utterance is a part of
                OR statement. This parameter is used only to simplify the
                conversation from MD story files. Don't use it other ways,
                because it ends up in a invalid story that cannot be user for
                real training.
```